explore-analyze/dashboards/add-controls.md

```diff
@@ -129,23 +92,37 @@ prompt control insertion for field values, field names, function configuration,
 apply to a specific panel, and exposes visualization configuration such as date histogram interval controls to dashboard users.
 
 :::{note}
-Currently, only **Options lists** with static values are supported for ES|QL-based controls.
+{applies_to}`stack: ga 9.1` {applies_to}`serverless: ga` Only **Options lists** are supported for ES|QL-based controls. Options can be static values, fields, functions, or values defined by a query.
+
+{applies_to}`stack: ga 9.0` Only **Options lists** with static values are supported for ES|QL-based controls.
```

Comment:
In 9.0 we also support values from a query. the biggest differences from 9.1 are:

-  we now support functions
- the users can create a control first from the controls dropdown

Reply:
Thanks for the details, I reworked this note in [af5068c](https://github.com/elastic/docs-content/pull/1917/commits/af5068cfbf1ce1607add0b56670e5b6378c08874)